unlock-app/src/createUnlockStore.js

```diff
@@ -15,7 +14,7 @@ import lockMiddleware from './middlewares/lockMiddleware'
 
 const config = configure(global)
 
-export default function createUnlockStore(defaultState = loadState()) {
```

Comment:
This did not work because `defaultState` was indeed defined and passed as the "brand new" default state as loaded in the HTML via server side rendering.